components/raftstore/src/store/fsm/apply.rs

```diff
@@ -758,7 +758,10 @@ fn should_write_to_engine(cmd: &RaftCmdRequest) -> bool {
             AdminCmdType::ComputeHash |
             // Merge needs to get the latest apply index.
             AdminCmdType::CommitMerge |
-            AdminCmdType::RollbackMerge => return true,
+            AdminCmdType::RollbackMerge |
+            // Flashback needs to get the latest region meta.
```

Comment:
no need now as we get the latest region meta from memory

Reply:
Fixed. PTAL.